src/features/organizations/hooks/useFollowOrgMutations.ts

```diff
@@ -11,11 +12,13 @@ export default function useFollowOrgMutations(orgId: number) {
       follow: true,
     });
     dispatch(orgFollowed(membership));
+    dispatch(allEventsUnload());
```

Comment:
Instead of dispatching a separate action, the `eventsSlice` could have listened for the `orgFollowed` and `orgUnfollowed` actions and reset the state then. There's good reasons for either solution, IMO.

If you want, and feel that reusing the same actions would be a nicer separation of concerns, you can submit a separate PR.